### PR TITLE
Settings: Update Chinese translation

### PR DESCRIPTION
**Pull request by NinelieZ**

### CODE REVIEW

**Comment by idoybh** [comment]
Please fix indentations and remove non translateable strings

**Comment by NinelieZ** [comment]
Sorry for my carelessness, I've fixed them, please check.

File: res/values-zh-rCN/yaap_configs.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+
+    <!-- Device specific doze package -->
+    <string name="config_customDozePackage" translatable="false"></string>
```
Comment (idoybh) [inline res/values-zh-rCN/yaap_configs.xml]:
```suggestion
```

File: res/values-zh-rCN/yaap_strings.xml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    <!-- Cycle through ringer modes gesture -->
+    <string name="prevent_ringing_option_cycle">循环通过模式</string>
+    <string name="prevent_ringing_option_cycle_summary">开启 (循环通过模式)</string>
+    <string name="prevent_ringing_option_normal">普通</string>
+
+    <!-- OTA App -->
+    <string name="update_settings_title">系统更新</string>
+    <string name="update_settings_summary">检查系统最新版本</string>
+    <string name="update_package" translatable="false">eu.chainfire.opendelta</string>
+    <string name="update_activity" translatable="false">eu.chainfire.opendelta.MainActivity</string>
```
Comment (idoybh) [inline res/values-zh-rCN/yaap_strings.xml]:
```suggestion
```

**Review by idoybh — CHANGES_REQUESTED** [review]

**Review by idoybh — APPROVED** [review]

**Comment by idoybh** [comment]
Thank you :)

**Pull request merged**